test/puppetlabs/puppetdb/cli/services_test.clj

```diff
@@ -15,18 +14,6 @@
             [clj-time.core :refer [now]]
             [puppetlabs.puppetdb.cli.export :as export :refer [facts-for-node]]))
 
-(deftest update-checking
-  (testing "should check for updates if running as puppetdb"
-    (with-redefs [v/update-info (constantly {:version "0.0.0" :newer true})]
-      (with-log-output log-output
-        (maybe-check-for-updates "puppetdb" "update-server!" {})
-        (is (= 1 (count (logs-matching #"Newer version 0.0.0 is available!" @log-output)))))))
-
-  (testing "should skip the update check if running as pe-puppetdb"
-    (with-log-output log-output
-      (maybe-check-for-updates "pe-puppetdb" "update-server!" {})
-      (is (= 1 (count (logs-matching #"Skipping update check on Puppet Enterprise" @log-output)))))))
```

Comment:
Well the check-for-updates test is irrelevant since we use the library for that now but I'll add the other test